### PR TITLE
FLA-1395 Added test for duplicate doc from parent app

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

[FLA-1395](https://justice.gov.bc.ca/jira/browse/FLA-1395)

Added regression tests to confirm documents with the same name uploaded via the parent app produces an error message on the Package Confirmation screen.

- refactored tests to accept multiple files instead of just one
- added a data-testId to frontend Toast to isolate and test the existence of the `<div />`, updated snapshots

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/helpers/PayloadHelper.java [hunk 1]
```diff
@@ -1,40 +1,53 @@
...
-                "                \"statutoryFeeAmount\": 0,\n" +
-                "                \"data\": {},\n" +
-                "                \"md5\": \"string\", \n" +
-                "                \"actionDocument\": {\n" +
-                "                   \"id\": 2,\n" +
-                "                   \"status\": \"" + actionStatus + "\",\n" +
-                "                   \"type\": \"F\"\n" +
-                "               }\n" +
-                "            }\n" +
+    	StringBuffer sb = new StringBuffer();
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply tests/src/test/java/ca/bc/gov/open/jag/efiling/helpers/PayloadHelper.java:12]: Invalid. StringBuilder is not thread safe.

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/helpers/PayloadHelper.java [hunk 2]
```diff
@@ -1,40 +1,53 @@
...
+		        "        },\n" +
+		        "        \"documents\": [\n");
+        
+        for (int i = 0; i < fileSpecs.length; i++) {
+            if (i > 0) {
+            	sb.append(",");
+            }
+            sb.append(
+	            "            {\n" +
+	            "                \"name\": \"" + fileSpecs[i].getFilename() + "\",\n" +
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply tests/src/test/java/ca/bc/gov/open/jag/efiling/helpers/PayloadHelper.java:37]: existing code for a static json payload.

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/helpers/PayloadHelper.java [hunk 3]
```diff
@@ -1,40 +1,53 @@
...
+	            "                \"type\": \"AFF\",\n" +
+	            "                \"statutoryFeeAmount\": 0,\n" +
+	            "                \"data\": {},\n" +
+	            "                \"md5\": \"string\", \n" +
+	            "                \"actionDocument\": {\n" +
+	            "                   \"id\": 2,\n" +
+	            "                   \"status\": \"" + fileSpecs[i].getActionStatus() + "\",\n" +
+	            "                   \"type\": \"F\"\n" +
+	            "               }\n" +
+	            "            }\n");
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply tests/src/test/java/ca/bc/gov/open/jag/efiling/helpers/PayloadHelper.java:47]: existing code for a static json payload.

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/helpers/PayloadHelper.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply tests/src/test/java/ca/bc/gov/open/jag/efiling/helpers/PayloadHelper.java:37]: existing code for a static json payload.

**Review by akroon3r — APPROVED** [review]
didn't see anything on first pass, but I'd definitely like a second set of eyes take a look 😄

**Pull request merged**